contracts/ReleaseDB.sol

```diff
@@ -54,12 +54,12 @@ contract ReleaseDB is Authorized {
    *  Modifiers
    */
   modifier onlyIfVersionExists(bytes32 versionHash) {
-    require(versionExists(versionHash));
+    require(versionExists(versionHash), "ethpm@releaseDB:version-does-not-exist");
```

Comment:
`missing-release-version`

Reply:
I think `version-not-found` and `release-not-found`?